### ISSUE TITLE
Python API注释优化需求

### ISSUE THREAD

**Issue by tink2123** [PaddlePaddle/Paddle#14478]
### 背景：

  PM文档方向于近期走查了Paddle API Reference文档（基于v1.1），总结共计78个python api注释中存在以下问题：

  1） 文字描述较少、抽象，很难理解
  2） 参数说明错误或不齐全
  3） 代码示例错误或不齐全
  4） 格式混乱

其中代码示例错误和格式混乱的问题共计42个，将由PM进行修复
剩余问题总结如下，辛苦研发老师支持修复

**由于API文档对用户使用体验影响较大，我们将提升此项工作的优先级，期望在v1.2前修复，并随v1.2上线**

###  一、文字描述较少、抽象，很难理解

1. [_switch_scope](https://github.com/PaddlePaddle/Paddle/blob/0abfbd1c41e6d558f76252854d4d78bef581b720/python/paddle/fluid/executor.py#L39):
   
   没有文档内容

2. [scope](https://github.com/PaddlePaddle/Paddle/blob/9f252e0032af38b064ba049993d8b59ccc9bce3a/paddle/fluid/framework/scope.h)
   
    没有文字描述内容，是否补充？
    ![default](https://user-images.githubusercontent.com/31891223/48685903-f7b1b900-ebf3-11e8-95cb-7b3b9ec3a49b.png)

3. [shuffle](https://github.com/PaddlePaddle/Paddle/blob/d3aed98d86af288a15047eee75ed8c939b0babb6/python/paddle/fluid/layers/io.py#L946)
    
    内容过于简略，缺少参数说明和代码示例
    ![](https://user-images.githubusercontent.com/31891223/48536300-20326e00-e8e9-11e8-8a08-5f64fbe656c1.png)

4. [ctc_greedy_decoder](https://github.com/PaddlePaddle/Paddle/blob/d3aed98d86af288a15047eee75ed8c939b0babb6/python/paddle/fluid/layers/nn.py#L4116)
    
    看了案例感觉不是很懂 output代表什么，希望优化文字描述
    ![](https://user-images.githubusercontent.com/31891223/48394112-34416880-e74d-11e8-865b-d1959be37507.png)

5. [force_init_on_cpu](https://github.com/PaddlePaddle/Paddle/blob/cffad81c1a4e787d7495bf6e5e3d35f08ebae967/python/paddle/fluid/initializer.py#L32)

    文字说明需要补充，该API如何应用呢？
    ![](https://user-images.githubusercontent.com/31891223/48398529-f5ff7580-e75b-11e8-9bcb-516a166db446.png)

### 二、参数说明错误或不齐全

1. [staticRNN](https://github.com/PaddlePaddle/Paddle/blob/61b4812f2fe8c0591323f9d60db69231d8933322/python/paddle/fluid/layers/control_flow.py#L429)

    参数说明不完整，且缺少代码示例
    
    ![](https://user-images.githubusercontent.com/31891223/48390389-bd9c6f00-e73c-11e8-988a-bc8a8d34307d.png)
    
   
2. [DynamicRNN](https://github.com/PaddlePaddle/Paddle/blob/61b4812f2fe8c0591323f9d60db69231d8933322/python/paddle/fluid/layers/control_flow.py#L1542)
 
    格式错误，导致参数说明有误
    ![](https://user-images.githubusercontent.com/31891223/48042193-7c93e000-e1bb-11e8-9a47-5780c7de4a48.png)

   block中的 class docstring 是否写错了呢

  ![图片](http://agroup-bos.cdn.bcebos.com/04e69be26fb4c2804e9a4fadb32d916e1245f7c9)

    
###  三、缺少代码示例
    
1. nn 图像相关
 [resize_bilinear](https://github.com/PaddlePaddle/Paddle/blob/d3aed98d86af288a15047eee75ed8c939b0babb6/python/paddle/fluid/layers/nn.py#L5801)
    
  
2. nn 激活函数相关
['elu','relu6','pow','stanh','hard_sigmoid','swish', 'prelu','brelu','leaky_relu','soft_relu'](https://github.com/PaddlePaddle/Paddle/blob/d3aed98d86af288a15047eee75ed8c939b0babb6/python/paddle/fluid/layers/nn.py#L6659)

3. nn 初始化相关
['uniform_random_batch_size_like','gaussian_random','sampling_id','gaussian_random_batch_size_like'](https://github.com/PaddlePaddle/Paddle/blob/d3aed98d86af288a15047eee75ed8c939b0babb6/python/paddle/fluid/layers/nn.py#L7215)

4. nn 数学操作相关
['sum','slice','shape','logical_and','logical_or','logical_xor','logical_not','clip','clip_by_norm','mean','mul','sigmoid_cross_entropy_with_logits','maxout'](https://github.com/PaddlePaddle/Paddle/blob/release/1.1/python/paddle/fluid/layers/nn.py#L7027)
    
5. detection 相关
[polygon_box_transform](https://github.com/PaddlePaddle/Paddle/blob/4a55fb5f5b8d177a61133afe7210561f796a7e32/python/paddle/fluid/layers/detection.py#L383)

6. 学习率调整相关
[append_LARS](https://github.com/PaddlePaddle/Paddle/blob/26200f2e420566cba3112ee725197a1c12c8682b/python/paddle/fluid/layers/learning_rate_scheduler.py#L310)
